cleansweep/plugins/committees/forms.py

```diff
@@ -53,6 +51,29 @@ def ensure_empty_slots(self, n=5):
         for i in range(n-empty_slots):
             self.roles.append_entry()
 
+
+class NewCommitteeForm(CommitteeForm):
+    """
+    Creates a form for adding a new committee structure.
+    """
+    level = StringField('Level')  # Level field at which the committee structure is getting created.
+
+    def __init__(self, place, level, *a, **kw):
+        super(NewCommitteeForm, self).__init__(place, *a, **kw)
+        self.level.data = level
+
+
+class EditCommitteeForm(CommitteeForm):
+    """
+    Edit form for editing committee structures. Extends CommitteeForm.
+    """
+    level = SelectField('Level', choices=[])  # Select field for place level selection.
+
+    def __init__(self, place, *a, **kw):
+        super(EditCommitteeForm, self).__init__(place, *a, **kw)
+        place_types = [place.type] + place.type.get_subtypes()
+        self.level.choices = [(t.short_name, t.name) for t in place_types]
+
```

Comment:
If you're sure that we won't be changing levels in committee structures then we don't.
In #122 you [said](https://github.com/AamAadmiParty/cleansweep/issues/122#issuecomment-142810296) it should be allowed only if there's no other structure at that level.